src/components/List/ListItem.tsx

```diff
@@ -85,6 +107,10 @@ class ListItem extends UIComponent<Extendable<IListItemProps>, any> {
     this.setState({ isHovering: false })
   }
 
+  componentDidUpdate() {
+    this.focusableItem.tryFocus(ReactDOM.findDOMNode(this.itemRef.current!) as HTMLElement)
```

Comment:
this is better to be rewritten the following way, as we will reduce amount of casts used by one:

```js
ReactDOM.findDOMNode(this.itemRef.current) as HTMLElement
```

Reply:
done